docs/technical_reference/flowsheets/ion_exchange.rst

```diff
@@ -0,0 +1,146 @@
+    keywords, respectively. It will also set the operating conditions for the IX unit process simulation. Specific variables fixed are in the sections below.
+
+3. Connecting and initializing individual unit models with ``initialize_system``: 
+    Calling this function will initialize all unit models on the flowsheet (``feed``, ``ion_exchange``, ``product``, and ``regen``), initialize 
+    the ``costing`` block, and propagate the arcs.
+
+4. Solving the model with ``solver``:
+    The solver object is returned by calling the WaterTAP function ``get_solver()``. After the model is built, specified, and initialized, 
+    the model has zero degrees of freedom and is solved with ``solver.solve(m)``. The termination condition of the solve is checked and, assuming the solve is optimal, 
+    continues with the optimization demonstration.
+    
+5. Optimally optimizing IX design with ``optimize_system``: 
+    Though not required, this function provides an example of optimizing the design of the IX system for a targeted effluent concentration
+    of 25 mg/L. The initial model demonstration resulted in an effluent concentration of 0.21 mg/L, so this optimization results in longer breakthrough time
+    and therefore, a less frequent regeneration schedule and lower operating costs. The optimization proceeds in four steps:
+        
+    * First, the function will set an ``Objective`` on the flowsheet to minimize the levelized cost of water (LCOW).
+    
+    .. math::
+        
+        LCOW_{Q} = \frac{f_{crf}   C_{cap,tot} + C_{op,tot}}{f_{util} Q}
+
+    * Then, the model fixes the effluent concentration of the IX model to 25 mg/L, propagates that concentration to the product block, and re-initializes the product block with the new targeted concentration.
+    * Next, three variables are unfixed on the ion exchange model to allow for the model to solve for the new conditions (``dimensionless_time``, ``number_columns``, and the ``bed_depth``).
+    * Finally, the model is solved for these new conditions.
+    
+    Note that stopping at this point will likely result in a non-integer value for ``number_columns``. This value also corresponds to an optimal value for both ``bed_depth`` and 
+    ``dimensionless_time``. The user can optionally ensure the ``number_columns`` is an integer by rounding the ``number_columns`` to an integer value after 
+    the initial optimization solve, fixing ``number_columns`` to that value, and then re-solving the model to obtain the final values for ``bed_depth`` and ``dimensionless_time``.
+    This is the approach taken in the demonstration file.
+        
+Degrees of Freedom
+------------------
+
+The degrees of freedom for the flowsheet can change depending on the configuration options specified during the build. 
+Specifics for other configurations options are available in the IX model documentation.
```

Comment:
For convenience, I'd also link "IX model documentation" here to the documentation. I know it's already linked above, but it can't hurt to have it linked more than once.